### PR TITLE
Add example of default guilds in app commands guide

### PR DESCRIPTION
**Pull request by Jonxslays**
### Summary

Adds examples for setting default guilds for app commands to the docs guide.
Examples include using bot constructor, plugin constructor, and command decorator.
This seems to be a common question in the server.

### Checklist

- [x] I have run `nox` and all the pipelines have passed.

### CODE REVIEW

File: docs/source/guides/application-commands.rst [hunk 1]
```diff
@@ -116,7 +124,13 @@ Setting default guilds for all commands at once can be done using the ``default_
...
+Default Guild Resolution Order
+==============================
+
+When handling default guilds, lightbulb will resolve them in the following order:
+
+- Command-specific default guilds (see :obj:`lightbulb.commands.base.CommandLike.guilds`)
+
+- Plugin-specific default guilds (see :obj:`lightbulb.plugins.Plugin.default_enabled_guilds`)
+
+- Global default guilds (see :obj:`lightbulb.app.BotApp.default_enabled_guilds`)
```
Comment (Jonxslays) [inline docs/source/guides/application-commands.rst:136]: Should this be BotApp default guilds?

Comment (tandemdude) [reply docs/source/guides/application-commands.rst:136]: I think so

**Comment by tandemdude** [comment]
May be useful to mention that if you set command guilds or plugin default enabled guilds to an empty tuple then the relevant command(s) will be global even if the BotApp instance has `default_enabled_guilds` specified

**Comment by Jonxslays** [comment]
yeah that's a good tip

**Pull request merged**